back/app/models/home_page.rb

```diff
@@ -10,27 +10,34 @@
 #  top_info_section_multiloc                :jsonb            not null
 #  bottom_info_section_enabled              :boolean          default(FALSE), not null
 #  bottom_info_section_multiloc             :jsonb            not null
-#  events_widget                            :boolean          default(FALSE), not null
+#  events_widget_enabled                    :boolean          default(FALSE), not null
 #  projects_enabled                         :boolean          default(TRUE), not null
 #  projects_header_multiloc                 :jsonb            not null
 #  banner_avatars_enabled                   :boolean          default(TRUE), not null
-#  customizable_homepage_banner             :boolean          default(TRUE), not null
+#  customizable_homepage_banner_enabled     :boolean          default(TRUE), not null
 #  banner_layout                            :string           default("full_width_banner_layout"), not null
 #  banner_signed_in_header_multiloc         :jsonb            not null
-#  cta_signed_in_text_multiloc              :jsonb            not null
-#  cta_signed_in_type                       :string           default("no_button"), not null
-#  cta_signed_in_url                        :string
+#  banner_cta_signed_in_text_multiloc       :jsonb            not null
+#  banner_cta_signed_in_type                :string           default("no_button"), not null
+#  banner_cta_signed_in_url                 :string
 #  banner_signed_out_header_multiloc        :jsonb            not null
 #  banner_signed_out_subheader_multiloc     :jsonb            not null
 #  banner_signed_out_header_overlay_color   :string
 #  banner_signed_out_header_overlay_opacity :integer
-#  cta_signed_out_text_multiloc             :jsonb            not null
-#  cta_signed_out_type                      :string           default("sign_up_button"), not null
-#  cta_signed_out_url                       :string
+#  banner_cta_signed_out_text_multiloc      :jsonb            not null
+#  banner_cta_signed_out_type               :string           default("sign_up_button"), not null
+#  banner_cta_signed_out_url                :string
 #  created_at                               :datetime         not null
 #  updated_at                               :datetime         not null
 #  header_bg                                :string
 #
+# We don't use the 'banner_' prefix for customizable_homepage_banner_enabled to maintain the
```

Comment:
I think you need to move this comment above the automated table structure from annotate_models, otherwise it will be gone the next time the annotations are being updated (next db migration). So line 3/4.